conjur/client.py

```diff
@@ -213,3 +213,22 @@ def update_policy_file(self, policy_name, policy_file):
         Replaces a file-based policy defined in the Conjur instance
         """
         return self._api.update_policy_file(policy_name, policy_file)
+
+    def rotate_anothers_api_key(self, resource, resource_to_rotate):
+        """
+        Rotates a API keys and returns new API key
+        """
+        return self._api.rotate_anothers_api_key(resource, resource_to_rotate)
```

Comment:
do we want to add a client side validation? (permission wise.) just asking

Reply:
this is a long winding road.
Overloaded server could benefit from such approach (PVWA does this to Vault), but I think this needs to be added as an explicit performance feature, across all out major clients, and only if it is proven to improve server performance significantly.

Reply:
Agreed ^ 
I don't think there is much we can do specifically for rotate_anothers_api_key beyond making sure the user exists but don't think it would be worth the effort here. We could for example, on `change-password` action have some validation that a password meets the Conjur password complexity and allow the user another opportunity to input their new password without failing the request (as we are doing now). I also think that this is a widespread effort that is not unique to this PR that requires more thought. I will open a ticket for this effort because it can be applied in a lot of palces.